packages/cspell-trie/tsconfig.json

```diff
@@ -8,7 +8,8 @@
 		"alwaysStrict": true,
         "declaration": true,
 		"noImplicitAny": true,
-		"noImplicitThis": true,
+        "noImplicitThis": true,
+        "resolveJsonModule": true,
```

Comment:
This might break the command line app because the paths are now different.

Reply:
Undid this change